* vertex_iterator::operator+(size_t) is ambiguous with boost::iterator_facade::operator+
--- gala-19bcf7614c428f2b27f005d154aa546f96f83c74/boost.h.orig	2018-11-09 13:44:07.000000000 +0100
+++ gala-19bcf7614c428f2b27f005d154aa546f96f83c74/boost.h	2019-05-12 14:54:45.483009816 +0200
@@ -259,7 +259,7 @@
 					a.advance(n);
 					return std::move(a);
 				}
-#else
+#elif 0
 			vertex_iterator operator+(size_t n) {
 				vertex_iterator a=*this;
 				a.advance(n);
